pages/people/[id]/index.tsx

```diff
@@ -34,7 +34,12 @@ const PersonPage = (): React.ReactElement => {
             {canViewArea(user) ? <WarningNotes id={personId} /> : <></>}
             <PersonDetails person={person} />
             <AllocatedWorkers person={person} />
-            {canViewArea(user) ? <Relationships id={personId} /> : <></>}
+            {user.hasDevPermissions ||
+            (user.hasAdultPermissions && person.contextFlag == 'A') ? (
```

Comment:
Also nit pick but we should probably do `===` equality here, not `==`

Reply:
Done ;)